### ISSUE TITLE
Sprint 78 plan - Public Websites

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#12523]
### 1. Harden homepage for prod 

**Launch**
- [x] #12546 

**"Create Account" block:**
  - [ ] #11692 (needs refinement)
  - [ ] (stretch; depends on drupal) #12532 ()

### 2. Header/footer redesign
- Awaiting info / green light for kickoff

### 3. Income Limits web app
**Design / Research:**

All of the following are blocked pending stakeholder review/verification:
- [ ] (Wes; carry-over) #12384
- [ ] (blocked on stakeholder input) #12256  (8, includes launching recruiting)
- [ ] (blocked on stakeholder input) #12341 (8) 
- [ ] (blocked on stakeholder input) #12248 (5)
- [ ] (blocked on stakeholder input; not ticketed) Midpoint review - Include LauraF
- [ ] (blocked on stakeholder input) #12340 (8)
- [ ] (blocked on stakeholder input) #12343 (5)

**Engineering:**
- [ ] (carry-over) #12348 (5)
- [ ] #12584 (needs refining)
- [ ] (stretch) #12402 (8)

### 4. Benefit Content Model Hardening 
- [ ] (assuming DaveP avail) #12423 (5)
- [ ] #12408 ()

### 5. PACT Act Wizard
- Pending movement from stakeholders; first ticket is ready for Cindy (research) 

### 6. Accordion header levels 
- [ ] (carry-over) #12342 (5)
- [ ] (carry-over) #12345 (5)
- [ ] (Laura F) #12352 (8)
- [ ] (stretch; depends on testing) #12346 (2)

### 7. Cerner API deprecation support
- [ ] (unblocked!) #9307 (5)

###  Other sprint priorities 
- [ ] (priority?) #12625 ()
- [ ] #12339 (needs refining)
- [ ] #12568 
- [ ] (hand off to Facilities) #12583

### BTSSS widget
- [ ] #12531 (Wes)
- [ ] (stretch, pending collab cycle timing) #12591 ()

### User research and UX pipeline development
- [ ] #12601 (5)
- [ ] #12602 ()
- [ ] #12632 ()
---

### Redirects
- [ ] 2 redirects coming

### Tech debt 
- [X] (in progress) #10307 (4 remain)

**Comment by jilladams** [comment]
@wesrowe propose we try to pick up / fix this bug: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12529

**Comment by jilladams** [comment]
Add https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12546

**Comment by jilladams** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/53460 - incoming redirect, once IA approves I'll move it to our repo

**Comment by jilladams** [comment]
Research project epic that needs tasks / may need to get started in this sprint: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12583

**Comment by jilladams** [comment]
Income limits business logic update for the API design: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12584

**Comment by jilladams** [comment]
@wesrowe a new ticket for Cindy re: analytics training: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12602

**Comment by jilladams** [comment]
redirect incoming at some point: https://github.com/department-of-veterans-affairs/va.gov-team/issues/53634

**Comment by jilladams** [comment]
ARs: 
* Update https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12591 with discovery findings / implementation notes - @jtmst
